components/HomeButton.js

```diff
@@ -1,59 +1,63 @@
 import React from 'react';
 import styled from 'styled-components/native';
-import { View, Image, StyleSheet, TouchableOpacity, Text } from 'react-native';
 
+import { View, Image, StyleSheet, TouchableOpacity, Text } from 'react-native';
 const styles = StyleSheet.create({
     styledImage: {
-        width: 169,
-        height: 207,
+        width: 175,
+        height: 220,
```

Comment:
Fixing the size here is probably the reason why the button stays the same size on different screens. Should probably use a `min-width` and `min-height` instead. (Should try testing different things on local)

Reply:
I'll try this out locally! Thank you for the suggestions!